data/settings.js

```diff
@@ -1344,14 +1419,14 @@ characters \`(strlen)\`, e.g. \`'*******'\`, or a textual description
 \`(text)\`, e.g. \`'Spam'\` or \`'Not Spam'\`.`
 	},
 
-	'sieve_virustest_text_value<X>': {
+	'sieve_virustest_text_value': {
 		tags: [ 'sieve', 'sieve-virustest' ],
 		plugin: 'sieve',
-		values: setting_types.STRING,
+		values: setting_types.STRLIST,
 		seealso: [ '[[link,sieve_virustest]]' ],
-		text: `
+		text: `FIXME
 When the [[setting,sieve_virustest_status_type]] setting is set to
-\`text\`, these settings specify that the spamtest test will match against
+\`text\`, this setting specifies that the spamtest test will match against
 the value \`<X>\` when the specified string is equal to the text (extracted)
```

Comment:
no more <X>. I guess that's why this is marked FIXME.

Reply:
fixed